mne/evoked.py

```diff
@@ -291,7 +291,8 @@ def save(self, fname):
         write_evokeds(fname, self)
 
     def __repr__(self):
-        s = "comment : '%s'" % self.comment
+        s = ('kind :  %s, ' % self.kind) if self.kind != 'average' else ''
```

Comment:
(but in that 1% of non-average instances, you really want to know so I put it first)